src/Ng/Elfie/ElfieCmd.cs

```diff
@@ -107,18 +107,18 @@ public string RunIndexer(string targetDirectory, string packageId, string packag
         /// <summary>
         /// Runs the Elfie.Merger.exe command line tool to create an ardb/txt file.
         /// </summary>
-        /// <param name="idxListFile"></param>
+        /// <param name="idxDirectory"></param>
         /// <returns>If successful, returns the path to the ardb file.
         /// If there were no files to index, returns null.
         /// If there was an error generating the ardb file, an exception is thrown.
         /// </returns>
-        public string RunMerger(string idxListFile, string outputDirectory)
+        public string RunMerger(string idxDirectory, string outputDirectory)
```

Comment:
I know this change to the Elfie merger is odd (switching from a list of idx files to a directory of idx files), but it's an Elfie'ism. If you pass the merger a list of idx files, it assumes the list is sorted and doesn't try to reorder the packages in the index. This is why we used to calculate the log2 value and sort the list. However, if you pass the merger a directory of idx files, it reads the download counts from the idx and sorts the packages accordingly. Since we're writing the download counts into the idx files, we need to pass a directory instead of a file list.